### ISSUE TITLE
Mass create Screwdriver pipelines

### ISSUE THREAD

**Issue by tkyi** [screwdriver-cd/screwdriver#400]
Since we are frequently moving/taking down databases and recreating them, it would be nice to have a script/tool to easily regenerate Screwdriver pipelines with secrets

- [x] semantic release for all node  repos
- [x] script for mass create all pipelines
  - [x] node repos with 2 secrets: NPM_TOKEN, GH_TOKEN
  - [x] node repos(screwdriver, UI) with more secrets
  - [x] go repos
  - [x] no-secret repos
- [x] GIT_KEY for all repos that need it
- [x] Test out script using beta tables
- [x] Create pipelines in production postgres tables

List of all the repos:
https://gist.github.com/tkyi/e5870adcd13163d16d236a4cb387a3b4

Script:
https://gist.github.com/minz1027/81f9c43a87fab3e1878166d5f3d36ebf

Problems:
1. Hard to automate the creation of `GIT_KEY` secret. Can we get rid of `GIT_KEY` and use `GITHUB_TOKEN` instead?

**Comment by stjohnjohnson** [comment]
Problem #1, yes please.  Who still uses GIT_KEY?

**Comment by minz1027** [comment]
@stjohnjohnson `ui`, `homepage`, `guide` and the three GO repos: `gitversion`, `launcher`, `log-service`.

**Comment by tkyi** [comment]
**Status:** Waiting on #399 before trying to recreate all the pipelines.

**Comment by tkyi** [comment]
Pipelines have been recreated successfully.

- [ ] [optional] move mass-create pipelines script to central tools repo
